decidim-core/lib/tasks/decidim_webpacker_tasks.rake

```diff
@@ -62,6 +62,11 @@ namespace :decidim do
     task upgrade: :environment do
```

Comment:
The PR didn't fix the original report, as the instructions that we have don't mention the `upgrade` task, but the `install`: https://github.com/decidim/decidim/blob/c584d4a40ff61bb232d026dc1cbe3b68a4cf2263/RELEASE_NOTES.md?plain=1#L116

We should move this change to the `install` task section or change the instructions to use the `update` task

Reply:
Added the following content to the release notes: 


By this time, you should have already ran the below commands. If you have not, please do it now. Running the below commands will purge any webapcker related binaries `bin/webpack` and `bin/webpack-dev-server` and will install the shakapacker binaries named `bin/webpacker` and `bin/webpacker-dev-server`.

```console
bundle update decidim
bin/rails decidim:upgrade
bin/rails db:migrate
```

Reply:
> By this time, you should have already ran the below commands. 

Yes, I've done it

> Running the below commands will purge any webapcker related binaries bin/webpack and bin/webpack-dev-server and will install the shakapacker binaries named bin/webpacker and bin/webpacker-dev-server.

That's not the behavior that I'm seeing locally. Just for completeness, these are all the commands that I'm running:

```bash
git checkout release/0.27-stable
bundle
bin/rake development_app

git checkout develop
asdf local ruby 3.1.1

cd development_app
bin/spring stop

# There are some errors regarding the locked dependencies
# In the Gemfile, so we remove locks from some gems as a workaround
sed -i -e 's;gem "puma", ".*";gem "puma";g' Gemfile
sed -i -e 's;gem "parallel_tests", ".*";gem "parallel_tests";g' Gemfile
sed -i -e 's;gem "spring", ".*";gem "spring";g' Gemfile
sed -i -e 's;gem "spring-watcher-listen", ".*";gem "spring-watcher-listen";g' Gemfile
sed -i -e 's;gem "faker", ".*";gem "faker";g' Gemfile
rm Gemfile.lock

bundle install
bundle update decidim
rm config/initializers/social_share_button.rb
bin/rails decidim:upgrade
```

After this last command, I still have the webpack related binstubs:                                                                                                                              

```bash
ls bin/
# rails  rake  setup  spring  webpack  webpack-dev-server
```

Then if I finish with the instructions from the RELEASE_NOTES, i have the error that I mentioned before:
          
```bash
bin/rails db:migrate
bundle exec rake decidim:webpacker:install
```

The error:

```
(...)
Run `npm audit` for details.
rake aborted!
Errno::ENOENT: No such file or directory @ rb_sysopen - /home/apereira/Work/decidim/decidim/development_app/bin/webpacker
/home/apereira/Work/decidim/decidim/decidim-core/lib/tasks/decidim_webpacker_tasks.rake:168:in `readlines'
/home/apereira/Work/decidim/decidim/decidim-core/lib/tasks/decidim_webpacker_tasks.rake:168:in `add_binstub_load_path'
/home/apereira/Work/decidim/decidim/decidim-core/lib/tasks/decidim_webpacker_tasks.rake:54:in `block (3 levels) in <main>'
/home/apereira/.asdf/installs/ruby/3.1.1/bin/bundle:25:in `load'
/home/apereira/.asdf/installs/ruby/3.1.1/bin/bundle:25:in `<main>'
Tasks: TOP => decidim:webpacker:install
(See full trace by running task with --trace)
(...)                        
```

Reply:
> 

Well, the test scenario may have a flaw in this command... 

```
git checkout develop
```
Maybe you should run : 

```
git remote add tremend https://github.com/tremend-cofe/decidim.git
git fetch tremend 
git checkout fix/10800
```

Then resume your commands: 

```
asdf local ruby 3.1.1

cd development_app
bin/spring stop

# There are some errors regarding the locked dependencies
# In the Gemfile, so we remove locks from some gems as a workaround
sed -i -e 's;gem "puma", ".*";gem "puma";g' Gemfile
sed -i -e 's;gem "parallel_tests", ".*";gem "parallel_tests";g' Gemfile
sed -i -e 's;gem "spring", ".*";gem "spring";g' Gemfile
sed -i -e 's;gem "spring-watcher-listen", ".*";gem "spring-watcher-listen";g' Gemfile
sed -i -e 's;gem "faker", ".*";gem "faker";g' Gemfile
rm Gemfile.lock

bundle install
bundle update decidim
rm config/initializers/social_share_button.rb
bin/rails decidim:upgrade
```

then you may be able to see an output like this: 
```
alecslupu@alecslupu-MS-7A33:~/Sites/tremend/dife/backend/decidim/development_app$ ls bin/
rails  rake  setup  spring  webpack  webpack-dev-server
alecslupu@alecslupu-MS-7A33:~/Sites/tremend/dife/backend/decidim/development_app$ rm config/initializers/social_share_button.rb
alecslupu@alecslupu-MS-7A33:~/Sites/tremend/dife/backend/decidim/development_app$ bin/rails decidim:upgrade
Copied migration 20230630074545_create_decidim_authorization_transfers.decidim.rb from decidim
Copied migration 20230630074546_rename_terms_of_use.decidim.rb from decidim
Copied migration 20230630074547_change_states_on_amendments.decidim.rb from decidim
Copied migration 20230630074548_change_states_on_reminder_records.decidim.rb from decidim
Copied migration 20230630074549_change_steps_dates_to_datetime.decidim_participatory_processes.rb from decidim_participatory_processes
Copied migration 20230630074550_change_states_on_collaborative_drafts.decidim_proposals.rb from decidim_proposals
Copied migration 20230630074551_change_states_on_proposals.decidim_proposals.rb from decidim_proposals
Copied migration 20230630074552_change_types_and_registration_types_on_meetings.decidim_meetings.rb from decidim_meetings
Copied migration 20230630074553_add_field_values_and_target_to_decidim_templates.decidim_templates.rb from decidim_templates
Copied migration 20230630074554_add_published_at_to_decidim_blogs_posts.decidim_blogs.rb from decidim_blogs
Copied migration 20230630074555_add_weight_to_conferences.decidim_conferences.rb from decidim_conferences
Copying binstubs
       exist  bin
      create  bin/webpacker
      create  bin/webpacker-dev-server
      create  bin/yarn
alecslupu@alecslupu-MS-7A33:~/Sites/tremend/dife/backend/decidim/development_app$ ls bin/
rails  rake  setup  spring  webpacker  webpacker-dev-server  yarn
alecslupu@alecslupu-MS-7A33:~/Sites/tremend/dife/backend/decidim/development_app$ 
```

Reply:
> Well, the test scenario may have a flaw in this command...
>
> ``` 
> git checkout develop
> ``` 

In my defense, I want to say that it was really early in the morning when I wrote that comment 🤣 

I probably made the same mistake yesterday too, by checking out the wrong branch. I see that it works as expected 👏🏽 👏🏽